src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage = () => {
+  const [peoples, setPeoples] = useState<Person[]>([]);
+  const [showLoader, setShowLoader] = useState(false);
```

Comment:
better put true by default and delete 13 line to avoid the appearing table with a switch between "Pages"